app/src/main/java/com/jvoye/tasky/core/database/mappers/RoomConverters.kt

```diff
@@ -0,0 +1,27 @@
+package com.jvoye.tasky.core.database.mappers
+
+import androidx.room.TypeConverter
+import com.jvoye.tasky.agenda.domain.TaskyType
+import com.jvoye.tasky.core.domain.model.Attendee
+
+class RoomConverters {
+    @TypeConverter
+    fun fromTaskyType(taskyType: TaskyType): String {
+        return taskyType.name
+    }
+
+    @TypeConverter
+    fun toTaskyType(value: String): TaskyType {
+        return TaskyType.valueOf(value)
+    }
+
+    @TypeConverter
+    fun fromAttendeeList(attendees: List<Attendee>): String {
+        return attendees.joinToString(",") { "${it.username},${it.email}" }
+    }
+
+    @TypeConverter
+    fun toAttendeeList(value: String): List<Attendee> {
+        TODO()
```

Comment:
We’ll need to finish out these converters to get the database compiling and working correctly. The enum mapping looks good, but the toAttendeeList still has a `TODO()`. Without implementing proper serialization/deserialization (JSON would be the easiest), Room won’t be able to restore the data you save. Completing this piece is essential for the schema to build cleanly.